core/quill.js

```diff
@@ -227,6 +234,11 @@ class Quill {
     this.setContents(delta, source);
   }
 
+  toggleEmpty(empty = false) {
```

Comment:
This function might not be necessary (see above comments). Even if it is necessary, it is not clear it should be a publicly exposed method.